gcam_to_tell/README.MD

```diff
@@ -25,41 +26,26 @@ TELL needs the following information:
+demand value for each year.
+* We should double check the units. GCAM seems to simulate demand in EJ, but the output of the GCAM reader is in TWh. We
+need to check to make sure the unit conversion is done correctly.
 
 ## Expected Output Data Format
-
-A single .csv file is the expected format of this data extraction effort. The csv should contain a heavily nested data structure with the following column names:
-
-
-|column|description|dtype| example |
-|----------------|--------------------------|-----|-------------------|
-| scenario       | name of gcam scenario             | str | rcp85cooler_ssp3          |
-| region         | gcam region name                  | str | USA                        |
-| subRegion      | state acronym                     | str | AL                         |
-| param          | parameter name                    | str | elec_heat_rate_BTUperkWh   |
-
-
-
-## Output Variable Details
-
-### elec_cap_usa_GW
-
-* **Description:** Electricity capacity by technology, vintage, subregion, and year. In the dispatch model, GCAM explicitly tracks capacity.  The "elec capacity by tech and vintage" query is this endogenous capacity.  Generation is a function of capacity within the model - this capacity is dispatched for each dispatch segment to determine generation. (Generation can't exceed capacity * hours in segment * segment capacity factor, but it can be less than that if the capacity is not fully utilized).  So technology capacity factors at each time step are dynamic, although there is a maximum they can't exceed. Previously, GCAM only tracked energy in the model, and used generation to back-calculate capacity. But that's no longer the case with the dispatch model.  Capacity is endogenous - calculated by the model and written to the database (albeit in 'EJ-capacity' units). The unit "EJ-capacity" is energy for the year at 100% capacity factor to represent capacity. This can be converted to other units by dividing by the number of hours in the year.
-* **gcam query:** "elec capacity by tech and vintage"
-* **Original units:** EJ-capacity
-* **Output units:** GW
-* **Calculation:**
-
-        1 EJ = 277777.7777777778 GWh per year
-
-        conv_EJ_to_GW = 277777.77777777778 / 8760
-
-
+A single .csv file is the expected format of this data extraction effort. The csv should contain (at a minimum) the 
+following column names:
+
+| Column | Description | dtype | Example |
+|:-:|:-:|:-:|:-:|
+| scenario  | Name of gcam scenario | str | rcp85cooler_ssp3         |
+| region    | GCAM region name      | str | USA                      |
+| subRegion | State acronym         | str | AL                       |
+| param     | Parameter name        | str | elecFinalBySecTWh        |
+| x         | Year                  | int | 1975                     |
+| units     | Variable units        | str | Final Electricity by Sector (TWh) |

```

Comment:
Maybe I'm missing up but I don't see any unit conversions to `TWh` in this script anymore and further below I see `aggregated['units'] = 'Final Energy by Sector (EJ)'`.

Lets clarify:  what were the output units we want?  Then if we really do want `TWh`, given GCAM will be giving results in EJ, we should apply the conversion in this python script.